SDK/simVis/GOG/Annotation.cpp

```diff
@@ -44,10 +44,9 @@ GogNodeInterface* TextAnnotation::deserialize(
   p.parseGeometry<Geometry>(conf);
   GogNodeInterface* rv = NULL;
   osgEarth::Annotation::LabelNode* label = NULL;
+  label = new osgEarth::Annotation::LabelNode(text, p.style_);
   if (nodeType == GOGNODE_GEOGRAPHIC)
-    label = new osgEarth::Annotation::LabelNode(mapNode, p.getMapPosition(), text, p.style_);
-  else
-    label = new osgEarth::Annotation::LabelNode(text, p.style_);
+    label->setPosition(p.getMapPosition());
```

Comment:
This LabelNode is no longer being set up with a mapnode. Is that intentional?

Reply:
will fix